src/components/project/ProjectDetails.vue

```diff
@@ -2,10 +2,9 @@
     div
```

Comment:
Compose your in components in views/project. Right now you are composing them in the components and just importing the composed component into views which doesn't make much sense. The views should be where you import your components and put them together not composing views in components then importing them as a component.